internal/controller/streams.go

```diff
@@ -506,6 +514,8 @@ func ConnectStreams(ctx context.Context, logger *zap.SugaredLogger, envMap Envir
 				bufferedGrpcSyncer: bufferedGrpcSyncer,
 			}
 
+			sm.findHubbleRelay(ctx, sm.streamClient.ciliumNamespace)
```

Comment:
Why are you calling this here just to ignore the returned value?
```suggestion
			ciliumFlowCollector = sm.findHubbleRelay(ctx, sm.streamClient.ciliumNamespace)
```